src/coreclr/src/ToolBox/superpmi/readme.md

```diff
@@ -0,0 +1,288 @@
+# Overview
+
+This directory (`src/coreclr/src/ToolBox/superpmi` in the GitHub
+https://github.com/dotnet/runtime repository) contains the SuperPMI
+tool used for testing the .NET just-in-time (JIT) compiler.
+
+## Purpose
+
+SuperPMI has two primary uses:
+1. Verification that a JIT code change doesn't cause any asserts.
+2. Finding test code where two JIT compilers generate different code, or
+verifying that the two compilers generate the same code.
+
+Case #1 is useful for doing quick regression checking when making a source
+code change to the JIT compiler. The process is: (a) make a JIT source code
+change, (b) run that newly built JIT through a SuperPMI run to verify no
+asserts have been introduced.
+
+Case #2 is useful for generating assembly language diffs, to help analyze the
+impact of a JIT code change.
+
+## SuperPMI architecture
+
+SuperPMI works in two phases: collection and playback.
+
+In the collection phase, the system is configured to collect SuperPMI data.
+Then, run any set of .NET managed programs. When these managed programs invoke the JIT
+compiler, SuperPMI gathers and captures all information passed between the
+JIT and its .NET host. This data is post-processed to remove essentially
```

Comment:
Do we call our VM/EE .NET host?

Reply:
That's a bit of an odd phrasing that was in the original readme.txt, that I didn't change.